src/components/dish/dish.jsx

```diff
@@ -0,0 +1,43 @@
+import styles from './dish.module.sass';
+import { useTheme } from '../theme-context/hooks';
+import classnames from 'classnames';
+import { DARK_THEME } from '../theme-context/constants';
+import { useAuthorization } from '../auth-context/hooks';
+import { Link, useLocation } from 'react-router-dom';
+import { MenuItemCartSection } from '../menu-item-cart-section/menu-item-cart-section.jsx';
+
+export const Dish = () => {
+    const { theme } = useTheme();
+    const location = useLocation();
+    const { item, restaurantId } = location.state;
+    const {
+        auth: { isAuthorized },
+    } = useAuthorization();
+
+    return (
+        <div className={classnames({
+            [styles.dark]: theme === DARK_THEME,
+        })}>
+            <div id={item.id} className={styles.dish}>
+                <div className={styles.inner}>
+                    <div className={styles.img}>
+                        <img src={'src/images/dish-d.jpg'} alt={item.name} />
+                    </div>
+                    <div>
+                        {item.name} - {item.price && <span>{item.price} USD</span>}
+                        <div className={styles.ingredients}>
+                            {Boolean(item.ingredients.length) &&
+                                item.ingredients.map((item) => (
+                                    <span>{item} </span>
```

Comment:
здесь нужно добавить key

Reply:
Пыталась исправить, но возникает ошибка
Warning: Encountered two children with the same key, `25402233-0095-49ea-9939-1e67ed89ffb9`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

Reply:
ключи должны быть уникальными, здесь по идее можно использовать текст самого ингредиента, так как в рамках блюда он уникален

Reply:
спасибо, поправила